tools/FileTestDefs.sh

```diff
@@ -38,9 +38,34 @@ staged_grep() {
 # Utility function for reporters that enables lots of decorators in grep
 # Works like staged_grep
```

Comment:
it would be helpful to have a comment here indicating the expected syntax for the function, specifically `pretty_grep [-flags...] [--] pattern filenames...`